public/app/features/dashboard/components/GenAI/utils.test.ts

```diff
@@ -8,6 +8,7 @@ import { getDashboardChanges, getPanelStrings, isLLMPluginEnabled, sanitizeReply
 
 // Mock the llms.openai module
 jest.mock('@grafana/experimental', () => ({
+  ...jest.requireActual('@grafana/experimental'),
```

Comment:
This is needed to prevent issues arising from importing and calling functions from `app/plugins/datasource/loki/queryUtils` in `public/app/features/trails/Integrations/logsIntegration.ts`. Somewhere, an indirect dependency on `@grafana/experimental` causes `GenAI/utils.test.ts` tests to fail unless we include the rest of the `@grafana/experimental` module in this mock.